config/config.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  development: {
+    use_env_variable: process.env.DEV_DATABASE_URL,
+    dialect: 'postgres',
+  },
+};
```

Comment:
This would throw an error subsequently( when you try to create a model). Sequelize by default expects a `config.json` file. For this use case you would need `.sequelizerc`